spec/order_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'order' 
+describe Order do
+  default_menu = Menu::MENU
+  let(:menu) { instance_double(Menu, dishes: default_menu) }
```

Comment:
I'd recommend creating a new menu for your tests, instead of using the default_menu here. Currently if a change happens to `Menu::MENU` it could break your `order_spec` tests, which means that they are not isolated properly.